puppet/.librarian/puppet/config

```diff
@@ -0,0 +1,3 @@
+---
+LIBRARIAN_PUPPET_DESTRUCTIVE: 'false'
```

Comment:
With destructive mode true, librarian-puppet will delete the entire `modules` directory before each install.